client/wildcard/src/components/Charts/components/bar-chart/BarChart.tsx

```diff
@@ -76,13 +78,12 @@ export function BarChart<Datum>(props: BarChartProps<Datum>): ReactElement {
         [categories]
     )
 
-    const yScale = useMemo(
-        () =>
-            scaleLinear<number>({
-                domain: [0, Math.max(...categories.map(category => category.maxValue))],
-            }),
-        [categories]
-    )
+    const yScale = useMemo(() => {
+        const highestValue = Math.max(...categories.map(category => category.maxValue))
+        return scaleLinear<number>({
+            domain: [0, Math.max(highestValue, maxValueLowerBound ?? -Infinity)],
+        })
+    }, [categories])
```

Comment:
I think the dependency array is missing `maxValueLowerBound`. Also there is a maximum number of parameters a function can take... if `categories` will ever be larger than that the call will fail. I don't know the maximum off the top of my head, and I would assume that we have other problems if `categories` will ever be that size, but who knows.

As an alternative, something like (needs testing)

```suggestion
    const yScale = useMemo(() => scaleLinear<number>({
        domain: [0, categories.reduce((max, category) => Math.max(max, category.maxValue), maxValueLowerBound ?? -Infinity)],
    }), [categories, maxValueLowerBound])
```

Reply:
Thank you!